Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -210,7 +330,15 @@ def main() -> None:
         elif demisto.command() == "arcanna-set-default-job-id":
             result_set_default_id = set_default_job_id(client, demisto.args())
             return_results(result_set_default_id)
-
+        elif demisto.command() == "arcanna-send-event-feedback":
+            result_send_feedback = send_event_feedback(client, feature_mapping, demisto.args())
+            return_results(result_send_feedback)
+        elif demisto.command() == "arcanna-send-bulk-events":
+            result_bulk = send_bulk_events(client, feature_mapping, demisto.args())
+            return_results(result_bulk)
+        elif demisto.command() == "arcanna-get-feedback-field":
+            result_feedback_field = get_feedback_field(demisto.params())
```

Comment:
You are using the `closing_reason_field` param only in this command. So if I get you, this should be a command argument and not an integration param. Please change it.

Reply:
In addition, please note that the command does not use the API, is it ok? if yes, please explain what is this command purpose.

Reply:
this command is used to populate on the ContextData `Arcanna.FeedbackField` which is used in the playbooks.
I could not find  a better way to get from an existing integration a value supplied as argumetnt

Reply:
any better solutions?